src/transformStateWithClones.js

```diff
@@ -7,7 +7,33 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const add = 'addProperties';
+  const remove = 'removeProperties';
+  const res = [];
+  const stateCopy = Object.assign({}, state);
```

Comment:
you can use spread syntax here instead
```suggestion
  const stateCopy = { ...state };
```